metatron/src/main/java/uk/gov/ida/eidas/metatron/domain/MetadataResolverService.java

```diff
@@ -78,8 +78,8 @@ public CountryMetadataResponse getCountryMetadataResponse(URI entityId) {
 
     private Client getClient(EidasCountryConfig country) {
         ClientConfig configuration = new ClientConfig();
-        configuration.property(ClientProperties.CONNECT_TIMEOUT, 10000);
-        configuration.property(ClientProperties.READ_TIMEOUT, 10000);
+        configuration.property(ClientProperties.CONNECT_TIMEOUT, 5_000);
```

Comment:
Is this mean't to be 5_000 or 5000 is this milliseconds?

Reply:
Yeah, 5_000 == 5000

It's a nice Java thing that you can add underscores to numbers like you would in writing like 5,000 and makes them a little more readable.

It is milliseconds also.

I briefly considered a constant like `TIMEOUT_MS = 5_000` but it seemed overkill.